evalai/management/commands/seed.py

```diff
@@ -0,0 +1,9 @@
+from django.core.management import BaseCommand
+
+
+class Command(BaseCommand):
+
+    help = "My custom django command"
```

Comment:
@TroJan yeah actually I was waiting for your review of `scripts/seed.py`. This is just a WIP code.